bemani/backend/popnhell/popnhello.py

```diff
@@ -0,0 +1,137 @@
+# vim: set fileencoding=utf-8
+import binascii
+import copy
+import base64
+from collections import Iterable
+from typing import Any, Dict, List, Sequence, Union
+
+from bemani.backend.popnhell.base import PopnHelloBase
+from bemani.backend.ess import EventLogHandler
+from bemani.common import ValidatedDict, GameConstants, VersionConstants, Time
+from bemani.data import UserID
+from bemani.protocol import Node
+
+class HelloPopnMusic(
+    EventLogHandler,
+    PopnHelloBase,
+):
+    name = "Hello! Pop'n Music"
+    version = VersionConstants.HELLO_POPN_MUSIC
+
+    @classmethod
+    def handle_game_common_request(self, request: Node) -> Node:
+        #game_common
+        root = Node.void('game')
+
+        flag = Node.void('flag')
+        root.add_child(flag)
+
+        flag.set_attribute(id, '1')
```

Comment:
Was this even tested? None of these variables are defined, this does nothing but crash?

Reply:
sent the wrong file. this will def break things.